src/templates/resources/footer.json

```diff
@@ -1482,7 +1482,7 @@
                 "text": "Lavora con noi"
             },
             {
-                "url": "/informazioni/driver-partner/",
+                "url": "http://riders.justeat.it/application",
```

Comment:
`http`? 😢

Reply:
Awaiting confirmation on the protocol...